pontoon/base/forms.py

```diff
@@ -256,6 +259,23 @@ class Meta:
             "bugzilla",
         )
 
+    def clean_github(self):
+        github_username = self.cleaned_data.get("github")
+
+        # Define GitHub's username pattern
+        github_username_pattern = r"^[a-zA-Z\d](?:[a-zA-Z\d]|-(?=[a-zA-Z\d])){0,38}$"
```

Comment:
Missed to submit this comment 🤦🏼 

Do we want to simplify the regex as `^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$` and pass `re.IGNORECASE` to `re.match()`?